foodnet/membership/models.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+
+
+class UserProfile(models.Model):
+    MALE = 'm'
+    FEMALE = 'f'
+    SEX_CHOICES = (
+        (FEMALE, 'female'),
+        (MALE, 'male')
+    )
+    
+    user = models.OneToOneField(User)
+    middlename = models.CharField(max_length=255)
+    adr1 = models.CharField(max_length=255)
+    adr2 = models.CharField(max_length=255)
+    streetno = models.CharField(max_length=255)
+    floor = models.CharField(max_length=255)
+    door = models.CharField(max_length=255)
+    zip = models.CharField(max_length=255)
```

Comment:
Avoid overwriting builtins, not even when you "know it won't break anything" - because it will at some point :) I would suggest `postal_code`.